virt/virt-4-12-release-notes.adoc

```diff
@@ -47,8 +47,10 @@ The SVVP Certification applies to:
 //CNV-18488 SVVP for 4.12
 
 //CNV-21611-2
-{VirtProductName} no longer uses the image:Operator_Icon-OpenShift_Virtualization-5.png[{VirtProductName},40,40] logo. {VirtProductName} is now represented by the image:virt-icon.svg[{VirtProductName},40,40] logo for versions 4.9 and later.
+* {VirtProductName} no longer uses the image:Operator_Icon-OpenShift_Virtualization-5.png[{VirtProductName},40,40] logo. {VirtProductName} is now represented by the image:virt-icon.svg[{VirtProductName},40,40] logo for versions 4.9 and later.
 
+//CNV-15830 Memory dump.
+* You can create a VM memory dump for forensic analysis by using the xref:../virt/virt-using-the-cli-tools.adoc#vm-memory-dump-commands_virt-using-the-cli-tools[`virtctl memory-dump` command].
```

Comment:
You can create a VM memory dump for forensic analysis by using the -> Forensic analysis of Virtual Machines is now supported through the creation of a VM memory dump using the

Reply:
IBM style guide: Use active voice whenever possible.

Reply:
The problem I have here is that the sentence on its own seems orphaned and it would feel better if first given context in the way of a subject. Perhaps this would be better:
Memory dumps for forensic analysis of Virtual Machines: You can create a VM memory dump by using ...

Reply:
This is our release note style. Please see current release notes. Because they can be very long, we try to keep the entries as concise as possible.